src/aliceVision/image/curves.cpp

```diff
@@ -0,0 +1,250 @@
+#include <cmath>
+
+#include "curves.hpp"
+
+
+namespace alicevision {
+    namespace image {
```

Comment:
```suggestion
namespace alicevision {
namespace image {
```
Normally a pass of clang-format should give you the right formatting style defined in the .clang-format file at the root of the project